spec/spec_helper.rb

```diff
@@ -72,13 +72,14 @@ def missing_methods
 end
 
 
-def configure(default_currency: nil, legacy_json_format: nil, legacy_deprecations: nil, legacy_default_currency: nil)
+def configure(default_currency: nil, legacy_json_format: nil, legacy_deprecations: nil, legacy_default_currency: nil, crypto_currencies: nil)
   old_config = Money.config
   Money.config = Money::Config.new.tap do |config|
     config.default_currency = default_currency if default_currency
     config.legacy_json_format! if legacy_json_format
     config.legacy_deprecations! if legacy_deprecations
     config.legacy_default_currency! if legacy_default_currency
+    config.crypto_currencies = true if crypto_currencies
```

Comment:
nit:

```suggestion
    config.crypto_currencies = crypto_currencies unless crypto_currencies.nil?
```

Reply:
Shouldn't this be a boolean and then just `config.crypto_currencies = crypto_currencies`?